sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosAsyncClientTest.java

```diff
@@ -25,6 +27,10 @@ public CosmosAsyncClientTest(CosmosClientBuilder clientBuilder) {
     }
 
     public final CosmosClientBuilder getClientBuilder() {
+        ImplementationBridgeHelpers.CosmosClientBuilderHelper.CosmosClientBuilderAccessor accessor =
+            ImplementationBridgeHelpers.CosmosClientBuilderHelper.getCosmosClientBuilderAccessor();
+        accessor.setCosmosClientApiType(this.clientBuilder, ApiType.TABLE);
```

Comment:
Please dont set it in base class, all the test will start running with this api type